lib/platform_requirements.rb

```diff
@@ -0,0 +1,10 @@
+#frozen_string_literal: true
+
+# workaround for zeitwerk, without wrapping code in module error:
+# expected file ../../neo4j-ruby-driver/lib/platform_requirements.rb to define constant PlatformRequirements, but didn't
+# is being thrown
```

Comment:
BTW this is how zeitwerk works. Either the file name matches the module/class name or the file has to be excluded from zeitwerk loading. So this is not a workaround and no explanation is needed.